<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExerciseWidget</class>
 <widget class="QWidget" name="ExerciseWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: #f8f8f8;
}
 QPushButton {
                       background: white;
                       border-radius: 8px;
                       border: 1px solid #e0e0e0;
                       padding: 15px;
                       margin: 5px;
                       text-align: left;
                       font-size: 16px;
                       color: #333;
					   text-align: center
                   }
                   QPushButton:hover {
                       background: #f5f5f5;
                       border-color: #bdbdbd;
                   }
                   QPushButton:pressed {
                       background: #e0e0e0;
                   }
                   QPushButton:checked {
                       background: #e3f2fd;
                       border-color: #90caf9;
                   }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,4">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>50</number>
   </property>
   <property name="topMargin">
    <number>30</number>
   </property>
   <property name="rightMargin">
    <number>50</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
				font-size: 18px;
                padding: 16px;
                margin: 8px;
                color: #333333;
            }</string>
       </property>
       <property name="text">
        <string>练习模块</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="button0">
       <property name="text">
        <string>02绘图模块练习模块</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button1">
       <property name="text">
        <string>01绘图模块练习模块</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button2">
       <property name="text">
        <string>03绘图模块练习模块</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button3">
       <property name="text">
        <string>01和02绘图模块练习模块</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button4">
       <property name="text">
        <string>01和03绘图模块练习模块</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
